src/main/java/pl/net/rogala/eventy/controller/EventController.java

```diff
@@ -43,33 +46,41 @@ public String showSingleEvent(@PathVariable String eventId, Authentication authe
         model.addAttribute("showCommentForm", showCommentForm);
         model.addAttribute("event", eventOptional.get());
         model.addAttribute("comments", eventService.getAllCommentsToEvent(Long.parseLong(eventId)));
-
+//        model.addAttribute("showUsers", eventService.showAllUsersAssignedToEvent(Long.parseLong(eventId)));
 
         return "event/showSingleEvent";
     }
+
+    @PostMapping("/event/{id}/addUserToEvent")
+    public String handleNewUserAssignedToEvent(@PathVariable String id,@RequestParam String eventId, Authentication authentication) {
+        eventService.assignedUseToEvent(Long.parseLong(id), authentication.getName());
+        return "redirect:/event/" + eventId;
```

Comment:
I think you should remove @RequestParam String eventId and you should replace in return statement eventId by id